### PR TITLE
"Claim all" confirmation

### PR DESCRIPTION
**Pull request by SKProCH**
Closes the https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6643
------------
image

### CODE REVIEW

**Review by mitchej123 — CHANGES_REQUESTED** [review]
Please make this an option; this would drive me nuts.

**Comment by SKProCH** [comment]
@mitchej123, done

File: src/main/java/betterquesting/handlers/ConfigHandler.java [hunk 1]
```diff
@@ -33,6 +33,8 @@ public static void initConfigs()
 
 		BQ_Settings.zoomInToCursor = config.getBoolean("Zoom in on cursor", Configuration.CATEGORY_GENERAL, true, "Zoom in on cursor. If false, zooms in on center of screen.");
 		BQ_Settings.zoomOutToCursor = config.getBoolean("Zoom out on cursor", Configuration.CATEGORY_GENERAL, true, "Zoom out on cursor. If false, zooms out on center of screen.");
+
+		BQ_Settings.claimAllConfirmation = config.getBoolean("Claim all requires confirmation", Configuration.CATEGORY_GENERAL, true, "Zoom out on cursor. If false, zooms out on center of screen.");
```
Comment (mitchej123) [inline src/main/java/betterquesting/handlers/ConfigHandler.java:37]: the last text needs to be updated

**Comment by mitchej123** [comment]
One small issue, looks good after that

File: src/main/resources/assets/betterquesting/lang/en_US.lang [hunk 2]
```diff
@@ -71,6 +71,8 @@ betterquesting.gui.remaining_lives=Remaining Lives: %s
 betterquesting.gui.full_lives=Lives Full
 betterquesting.gui.closing_warning=Are you sure you want to close this editor?
 betterquesting.gui.closing_confirm=Changes may be lost!
+betterquesting.gui.claim_all_warning=Are you sure you want to claim all rewards?
+betterquesting.gui.claim_all_confirm=There can be a lot of items here that will drop to the ground if the inventory is full!
```
Comment (Prometheus0000) [inline src/main/resources/assets/betterquesting/lang/en_US.lang:75]: > There can be a lot of items here that will drop to the ground if the inventory is full!

There may be a lot of items that will drop to the ground if your inventory is full!

File: src/main/java/betterquesting/handlers/ConfigHandler.java (hunk 1 above)
Comment (SKProCH) [reply src/main/java/betterquesting/handlers/ConfigHandler.java]: Yes, I forgot about it

**Review by mitchej123 — APPROVED** [review]

**Pull request merged**